subgraphs/collectible-market/package.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "collectible-market",
```

```suggestion
  "name": "nft-market",
```
Pick one, and apply it everywhere.